Platform/TigerlakeBoardPkg/CfgData/CfgData_Silicon.yaml

```diff
@@ -278,7 +278,7 @@
       help         : >
                      0- Disable, 1- Enable (Default) Azalia controller
       length       : 0x01
-      value        : 0x01
+      value        : 0x00

```

Comment:
Is the intention to disable PchHdaEnable by default for all boards ?  
Or we just want to disable this for certain board ?  
If it is for specific  board,  we can do it in DLT file.

Reply:
Okey, disable TGL RVP board by default.